src/components/App.css

```diff
@@ -1,49 +1,6 @@
 .App {
   height: 100%;
-  max-width: 1000px;
+  max-width: 700px;
   min-width: 380px;
   width: 100%;
 }
```

Comment:
bring it in the middle for centered display
```
    margin: auto;
```

Reply:
implemented change on next milestone, thanks for the tip :+1: